File: AtCoderBeginnersSelection/ABC086A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int a, b;
    int product;
    cin >> a >> b;
    product = a * b;
    if (product % 2 == 0)
    {
        cout << "Even";
    }
    else
    {
        cout << "Odd";
    }
    return 0;
}

File: AtCoderBeginnersSelection/ABC086A/main.dart
import 'dart:io';

void main() {
  final input = stdin.readLineSync()!.split(' ');
  final a = int.parse(input[0]);
  final b = int.parse(input[1]);
  
  if ((a * b) % 2 == 0) {
    print('Even');
  } else {
    print('Odd');
  }
}
